main.tf

```diff
@@ -318,15 +320,14 @@ resource "aws_s3_bucket" "origin" {
   dynamic "website" {
     for_each = var.website_enabled ? local.website_config[var.redirect_all_requests_to == "" ? "default" : "redirect_all"] : []
     content {
-      error_document           = lookup(website.value, "error_document", null)
-      index_document           = lookup(website.value, "index_document", null)
-      redirect_all_requests_to = lookup(website.value, "redirect_all_requests_to", null)
-      routing_rules            = lookup(website.value, "routing_rules", null)
+      error_document           = website.value.error_document
+      index_document           = website.value.index_document
+      redirect_all_requests_to = website.value.redirect_all_requests_to
+      routing_rules            = website.value.routing_rules
```

Comment:
This causes a regression because these values are not sourced directly from `vars`, but from incomplete object definition(s) in locals here: https://github.com/jwadolowski/terraform-aws-cloudfront-s3-cdn/blob/9a1d533e852fa24a777006f6f5fb212bf2e79c23/main.tf#L73-L86

This results in the regression identified in #354